mypyc/ops.py

```diff
@@ -405,6 +405,29 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_call(self)
 
 
+class PyCall(RegisterOp):
+    """Python call f(arg, ...).
+    
+    All registers must be unboxed.
+    """
+    def __init__(self, dest: Optional[Register], function: Register, args: List[Register]) -> None:
+        self.dest = dest
+        self.function = function
+        self.args = args
+
+    def to_str(self, env: Environment) -> str:
+        args = ', '.join(env.format('%r', arg) for arg in self.args)
+        s = env.format('py(%r(%s))', self.function, args)
```

Comment:
What about using `x = f(arg, ...) :: py` as the str format (see my other comment)?